options.json

```diff
@@ -14,9 +14,6 @@
     "insertAt": {
       "type": ["string", "object"]
     },
-    "insertInto": {
-      "type": "string"
```

Comment:
Looks like we need new keyword in ajv for this

Reply:
Yep, a newer version of `schema-utils` is needed